### DISCUSSION TITLE
Add a way to embed the program right into Godot's main window in the main panel area

### DISCUSSION THREAD

**Discussion by filipworksdev** [pixelpen-dev/pixelpen#22] [Ideas]
I would prefer not to leave Godot since I have it set to fullscreen so it would be helpful if there was an option to add the program right into Godot's main panel like 2D, 3D , Script etc.

**Comment by bayu-sw** [comment]
It's possible, but the problem is PixelPen can't use keyboard shortcut that already being use by godot editor like CTRL+S, CTRL+Z and many more. By using window we can capture input before godot editor use it. If you just want to make godot editor fullscreen and keep PixelPen in the same screen simply toggle `Editor Setting > Interface > Editor > Single window mode (ENABLE)`. PixelPen window will be embeded inside godot editor.
